wrapanapi/systems/openstack.py

```diff
@@ -1255,3 +1255,23 @@ def download_object(self, container_name, object_name, path):
 
         with open(path, "wb") as obj:
             obj.write(obj_contents)
+
+    def get_quota(self, **kwargs):
```

Comment:
I think `project` and `quota` should be named kwargs, with defaults.

```
def get_quota(self, quota='all', project=None):
    quota = str(quota).lower()
    project = project or self.tenant
```

Reply:
Done